pkg/apis/sources/v1/sinkbinding_lifecycle.go

```diff
@@ -94,6 +94,15 @@ func (sbs *SinkBindingStatus) MarkSink(uri *apis.URL) {
 	}
 }
 
+func (sbs *SinkBindingStatus) MarkSinkCACerts(certs *string) {
+	sbs.SinkCACerts = certs
+	if certs != nil {
+		sbCondSet.Manage(sbs).MarkTrue(SinkBindingConditionSinkCACertsProvided)
+	} else {
+		sbCondSet.Manage(sbs).MarkFalse(SinkBindingConditionSinkCACertsProvided, "SinkCACertsEmpty", "Sink CA Certs has resolved to empty.%s", "")
+	}
+}
```

Comment:
Instead of using a new condition we can make use of the same `SinkBindingConditionSinkProvided`, see https://github.com/knative/eventing/pull/6929/files, we could update `MarkSink(uri *apis.URL)` to accept `Addressable` (aka `MarkSink(uri *duckv1.Addressable)`).

The main reason being CA certs resolving and Sink address resolving are not different or separate steps, so it's not incredibly interesting to have a `SinkCACertsProvided` condition in the status